stripe/res/layout/activity_payment_sheet.xml

```diff
@@ -5,76 +5,74 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-    <androidx.constraintlayout.widget.ConstraintLayout
+    <LinearLayout
```

Comment:
You'll need to update `stripe_activity_payment_options.xml` as well, right?

Reply:
Yes, it's changed here too